docs/src/configuration.md

```diff
@@ -80,8 +80,9 @@ The following implementations should work:
 - [Intel MPI](https://software.intel.com/en-us/mpi-library)
 - [Microsoft MPI](https://docs.microsoft.com/en-us/message-passing-interface/microsoft-mpi)
 - [IBM Spectrum MPI](https://www.ibm.com/us-en/marketplace/spectrum-mpi)
-
-If the implementation is changed, you will need to re-run `Pkg.build("MPI")`.
+- [MVAPICH](http://mvapich.cse.ohio-state.edu/)
+- [Cray MPICH](https://docs.nersc.gov/development/compilers/wrappers/)
+- [Fujitsu MPI](https://www.fujitsu.com/global/about/resources/publications/technicalreview/2020-03/article07.html#cap-03)
```

Comment:
I couldn't find better links for Cray MPICH and Fujitsu MPI (the Cray one is taken from https://github.com/spack/spack/blob/3835ac0c41ce57dda16de98c40b69dac5d26d34a/var/spack/repos/builtin/packages/cray-mpich/package.py#L16).  The line

>  If the implementation is changed, you will need to re-run `Pkg.build("MPI")`.

can probably be replaced with a reference to the section about using `MPIPreferences.jl` after #566 is merged.

Reply:
Can you open the PR against #566 ?

Reply:
Done!